README.md

```diff
@@ -61,36 +61,38 @@ exports.task = function* (ctx) {
 
 ## Overview
 
-`egg-schedule` is for time-based scheduling, not interval-based scheduling.  
+`egg-schedule` supports both time-based scheduling and interval-based scheduling.
 
-Just put your jobs at `{app_root}/app/schedule`, one file as one job.  
-
-Job file should export `schedule` and `task` properties.
+To setup a schedule task, simply create a job file in `{app_root}/app/schedule`. A file contains one job and should export `schedule` and `task` properties. The rule of thumbs is one job per file.
 
 ## Task
-Task should be a generator function, and accept ctx as params.
+
+Task is a generator function, and accept one parameter, `ctx`. ex: `exports.task = function* (ctx) { ... };`
 
 You can find schedule log at log file, which act like worker request, and contain:
```

Comment:
@mattma  
- egg will start one `master` process, cpu count `worker` process, one `agent` process
- `egg-schedule` will schedule at `agent` process,  but execute at `worker` process (one or all due to schedule type) 
- so when trigger, `agent` will send a message to `worker`, then the worker treat it just like a normal user http request. then this fake user request is use `SCHEDULE` as http method, `/__schedule?path=${s}&${qs.stringify(schedule.schedule)}` as url property.
- see https://github.com/eggjs/egg-schedule/blob/master/app.js#L29